rclcpp/include/rclcpp/time.hpp

```diff
@@ -102,6 +102,11 @@ class Time
   rcl_time_point_value_t
   nanoseconds() const;
 
+  /// \return the seconds since epoch as a floating point number.
```

Comment:
Should the docblock contain a note/warning about potential rounding errors? I think that was one reason to omit the function until now.

Reply:
Added warning in 7b736c955cddc636c2bf57e16d8c8353a02ae225